---
title: Analysis Of Long-Distance Linked Selection In Indian Variants Of SARS-CoV-2
summary: Mutations that thrive in the population confer survival advantage to the (viral) species. We attempted to explain the synergistic driving forces in long-distance mutations using association rule mining.
tags:
- Machine Learning
- Other
date: "2020-09-01"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: 
  focal_point: Smart

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
date_end: ""
show_date: false

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---

**Status**: Completed (June 2020 - Sept 2020)


This project was done virtually during the fist wave of the COVID-19 pandemic, in collaboration with [Prof. Arun Sethuraman](http://arunsethuraman.weebly.com/), California State University, San Marcos. Long range mutations are not well studied in evolutionary biology. We hypothesized that associations in long-range mutations gives more reasons to inspect causality in SARS-CoV-2. This would help to analyze the trajectory of mutations of the virus. We analyzed 1,200 full genome sequences and identified 14 significant non-synonymous mutations. The association rules were mined using apriori algorithm and their individual stability was analyzed using homology modelling. This was the first ever data-driven analysis of viral long-range mutations. We found out two clusters of significant non-synonymous mutations and analyzed the intra and inter-cluster rules (figure above). We presented our work at the virtual poster presentation at COVID-19 Dynamics and Evolution Virtual Conference, organised by UC San Diego School of Medicine.