---
title: Novel de-novo Short-Read Genome Assembler
summary: Genome assembly is an NP-hard variant of string reconstruction problem. We are builiding a novel de-novo short-read genome assembler which can be parallelized to reconstruct the DNA contigs from short reads efficiently.
tags:
- Other
date: "2021-01-01"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: 
  focal_point: Smart

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
date_end: ""
show_date: false

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---

**Status**: Ongoing (Since Jan 2021)

I am working as a student researcher at [Advanced Data Analytics and Parallel Technologies (ADAPT) Lab, BITS Pilani](https://www.bits-pilani.ac.in/pilani/computerscience/AdvancedDataAnalyticsParallelTechnologiesLaboratory) with [Prof. Poonam Goyal](https://universe.bits-pilani.ac.in/pilani/poonam/profile). We are working on biological [sequence assemby](https://en.wikipedia.org/wiki/Sequence_assembly) which is an NP-hard string reconstruction problem. We are building a novel de-novo short-read genome assembler to reconstruct the DNA contigs efficiently. The proposed assembler is being built with distributed and hierarchical data structures and using intermediaries- unitigs to avoid the bottlenecks in the pipeline. We harness overlap and paired-end information along with some novel features to extend the unitigs.

